dbt_coves/templates/source_model_props.yml

```diff
@@ -14,10 +14,9 @@ models:
   - name: {{ model.lower() }}
     columns:
 {%- for cols in nested.values() %}
-  {%- for col in cols %}
-      - name: {{ col.lower().replace(" ","_").replace(":","_").replace("(","_").replace(")","_") }}
-  {%- endfor %}
+      - name: {{ cols["key"].lower().replace(" ","_").replace(":","_").replace("(","_").replace(")","_") }}
+      - description: {{ cols["description"].lower().replace(" ","_").replace(":","_").replace("(","_").replace(")","_") }}
```

Comment:
@LeonardoPelanda if there is no description or description is empty, an if statement is needed to avoid writing the description line as in here:

models:
  - name: _airbyte_raw_country_populations
    columns:
      - name: country_code
         description:
      - name: country_name
         description:
      - name: value
         description: